types/theo/index.d.ts

```diff
@@ -43,7 +49,7 @@ export type ValueTransform =
 export function convert(options: ConvertOptions): Promise<string>;
 export function convertSync(options: ConvertOptions): string;
 export function registerFormat(
-    name: Format,
+    name: string,
```

Comment:
Good suggestion to open this up for custom formats 👍

Reply:
Wonder if we'd want to loosen up on the `Transform` and `ValueTransform` types to make customization easier as well.

Reply:
I was thinking about this too. I could do this too while at it. :-)